monkey/monkey_island/cc/services/initialize.py

```diff
@@ -40,4 +44,24 @@ def initialize_services(data_dir: Path) -> DIContainer:
 
 def _build_agent_binary_repository():
     file_repository = LocalStorageFileRepository(AGENT_BINARIES_PATH)
-    return AgentBinaryRepository(file_repository)
+    agent_binary_repository = AgentBinaryRepository(file_repository)
+
+    _log_agent_binary_hashes(agent_binary_repository)
+
+    return agent_binary_repository
+
+
+def _log_agent_binary_hashes(agent_binary_repository: IAgentBinaryRepository):
+    """
+    Logs all the hashes of the agent executables for debbuging ease
+
+    :param agent_binary_repository: Used to retrieve the agent binaries
+    """
+    AGENT_BINARIES = {
+        "Linux": agent_binary_repository.get_linux_binary(),
+        "Windows": agent_binary_repository.get_windows_binary(),
+    }
```

Comment:
We should handle `AgentRetrievalError`. Sometimes I may run with a binary missing because I don't need it for the test. The old logic handled these cases.